frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/currentUserComment/currentUserComment.component.tsx

```diff
@@ -43,38 +42,30 @@ export const CurrentUserComment = ({
 	message,
 	metadata,
 	images,
+	view,
 	onDelete,
 	onReply,
-	onEdit,
 	...props
 }: CurrentUserCommentProps) => {
 	const [isEditMode, setIsEditMode] = useState(false);
 	const readOnly = TicketsCardHooksSelectors.selectReadOnly();
-
-	const onEditImage = (img, index) => {
-		const newImages = [...images];
-		newImages[index] = img;
-		onEdit(_id, message, newImages);
-	};
-
-	// @ts-ignore
-	const onDeleteImage = (index) => onEdit(_id, message, images.toSpliced(index, 1));
-
-	const onUploadImages = async () => uploadImages((imagesToUpload) => onEdit(_id, message, images.concat(imagesToUpload)));
-	const imagesEditingFunctions = { onDeleteImage, onUploadImages, onEditImage };
+	const [commentReply, setCommentReply] = useState(metadata);
 
 	if (deleted) return (<DeletedComment author={author} />);
 
 	if (isEditMode) {
 		return (
 			<EditComment
-				message={message}
+				commentId={_id}
```

Comment:
Note: some viewpoints were saved the incorrect way but I believe this is because the branch was branched off a buggy staging version which viewpoints didnt work correctly. 

**To review after the branch gets updated.**